src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js

```diff
@@ -115,21 +130,27 @@ function showVisibilityCheckboxesIfCustomOptionSelected($containingForm) {
 }
 
 var checkCheckboxCallback = function(index, checkbox) {
-    if (checkbox.checked === false) {
+    var isChanged = checkbox.checked === false;
+    checkbox.checked = true;
+
+    if (isChanged) {
         $(checkbox).trigger('change');
     }
-    checkbox.checked = true;
 };
 
 var uncheckCheckboxCallback = function(index, checkbox) {
-    if (checkbox.checked === true) {
+    var isChanged = checkbox.checked === true;
+    checkbox.checked = false;
+
+    if (isChanged) {
         $(checkbox).trigger('change');
     }
-    checkbox.checked = false;
```

Comment:
The subsequent changes make it look like the callback variables (both check and uncheck) are not necessary anymore?